File: 02_swap_numbers/SwapNumbers.java
import java.util.*;

public class SwapNumbers {
    public static void main(String[] args){
        Scanner scan = new Scanner(System.in);
        int a = scan.nextInt();
        int b = scan.nextInt();
        System.out.println("Before Swapping: "+ a +" "+ b);
        a = a+b;
        b = a-b;
        a = a-b;
        System.out.println("After Swapping: "+ a +" "+ b);
        scan.close();
    }  
}

File: 02_swap_numbers/swap_numbers.cpp
#include <iostream>
using namespace std;

int main(){
    int a,b;
    cin>>a;
    cin>>b;
    cout<< "Before swapping: "<< a << " " << b << endl;
    a = a+b;
    b = a-b;
    a = a-b;
    cout<< "After swapping: "<< a << " " << b << endl;
    return 0;
}